src/main/java/edu/kit/ipd/crowdcontrol/objectservice/mail/MailHandler.java

```diff
@@ -39,22 +40,39 @@ public MailHandler(Properties props, Authenticator auth) throws MessagingExcepti
         connectToStore();
```

Comment:
the method returns a Message[] Array, you cant even read the messages without connected store. So the user of the messages has to open the store via message[i].getFolder(),getStore().open() before he can read that, and he has to close it after use. I can implement that this way and write it to the javadocs, but its a bit uncomfortable